src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -411,6 +411,22 @@ private BoundExpression BindSimpleBinaryOperator(BinaryExpressionSyntax node, Di
             int compoundStringLength = 0;
 
             BoundExpression result = BindExpression(current, diagnostics);
+
+            if (node.IsKind(SyntaxKind.SubtractExpression))
+            {
+                if (result.Kind == BoundKind.TypeExpression
+                    && !((current as ParenthesizedExpressionSyntax)?.Expression is ParenthesizedExpressionSyntax))
```

Comment:
Also, please move this check into the ```if (node.IsKind(SyntaxKind.SubtractExpression))``` as well.  

---
In reply to: [88722181](https://github.com/dotnet/roslyn/pull/15291#discussion_r88722181) [](ancestors = 88722181)